src/petals/utils/peft.py

```diff
@@ -118,6 +119,47 @@ def load_peft(
             time.sleep(delay)
 
 
+class GlobalAdapterMixin:
+    """A mixin that makes LoRA-wrapped linear layers obey a globally set adapter"""
+
+    ADAPTER_NOT_SET = "__ADAPTER_NOT_SET"
+    GLOBAL_ACTIVE_ADAPTER = ADAPTER_NOT_SET
```

Comment:
```suggestion
    global_active_adapter = None
```

1. `GLOBAL_ACTIVE_ADAPTER` is not a constant, it should not be in caps
2. Why use a predefined string constant instead of just `None`?